src/dymaptic.GeoBlazor.Core/Components/Geometries/Geometry.cs

```diff
@@ -11,20 +11,6 @@ namespace dymaptic.GeoBlazor.Core.Components.Geometries;
 [JsonConverter(typeof(GeometryConverter))]
 public class Geometry : MapComponent
 {
-    /// <summary>
```

Comment:
How come these are going away?

Reply:
As we discussed on Discord, these were non-functional, so should not have been included. Setting them did nothing, and they would always return `null`. I will see if I can implement their real functionality and add them back in soon.